src/calculateRentalCost.js

```diff
@@ -28,12 +28,16 @@
  */
 
 function calculateRentalCost(days) {
+  const rent = 40;
+  const off3 = 20;
+  const off7 = 50;
```

Comment:
don't use numbers in variables name
off3 and off7 are not a good name for the variables